GitUI/WindowPositionList.cs

```diff
@@ -44,17 +45,9 @@ public class WindowPositionList
         private static readonly string AppDataDir = Path.Combine(
             Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData), "GitExtensions");
 
-        private static readonly string ConfigFilePath = Path.Combine(AppDataDir, "WindowPositions.xml");
+        private static readonly string ConfigFilePath = Path.Combine(AppSettings.LocalApplicationDataPath.Value, "WindowPositions.xml");
```

Comment:
This will cause a reset of all windows after an upgrade... 🤔 
May not seem like a big deal but could upset some users.

Can we/how hard is it to load from local settings, if it exists, and fallback to load from roaming?
All changes should be written to the local folder though.

Reply:
The resulting path is the same as before, it is just constructed differently.

So there should be no reset of window positions.

Reply:
Good, thank you for the confirmation.